src/gapi/errors.py

```diff
@@ -1,12 +1,12 @@
 """GAPI and OAuth Token related errors methods."""
 
+from enum import Enum
 import json
+#import googleapiclient.errors
```

Comment:
this is breaking unittests:

https://travis-ci.org/jay0lee/GAM/jobs/631580165#L407

Reply:
if it's unused then the better solution would be to fix the unittests. @ejochman any ideas here?

Reply:
for now I've modiified the tests so they should pass without the unused import here. Let's go ahead and remove it for now and @ejochman can let us know if this is voiding the tests.